sonic-slave-stretch/Dockerfile.j2

```diff
@@ -249,6 +249,19 @@ RUN apt-get update && apt-get install -y \
         cmake \
 # For pam_tacplus build
         autoconf-archive \
+{%- if CONFIGURED_ARCH == "armhf" %}
+# For iproute2
+        cm-super-minimal \
+        libatm1-dev \
+        libelf-dev \
+        libmnl-dev \
+        libselinux1-dev \
+        linuxdoc-tools \
+        lynx \
+        texlive-latex-extra \
+        texlive-latex-recommended \
+        iproute2 \
+{%- endif %}
```

Comment:
there is no harm to make it for all platforms.

Reply:
Removed.